test/Microsoft.AspNetCore.Mvc.Core.TestCommon/LinkBuilder.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    public class LinkBuilder
```

Comment:
Is this just the thing we were already using in functional tests?

Reply:
Yeah